Sources/PointFree/EnvVars.swift

```diff
@@ -20,8 +20,9 @@ public struct EnvVars: Codable {
 
   public enum AppEnv: String, Codable {
     case development
-    case staging
     case production
+    case staging
+    case testing
```

Comment:
added a `testing` case so that we could pretty print only for tests. we could prob also get rid of staging.